src/v0/destinations/impact/util.js

```diff
@@ -92,8 +93,16 @@ const populateProductProperties = (productsMapping, properties) => {
         item[getProductsMapping(productsMapping, 'ItemQuantity')];
       productProperties[getPropertyName('ItemSku', index + 1)] =
         item[getProductsMapping(productsMapping, 'ItemSku')];
+
+      // Following product properties are build from configuration in RudderStack dashboard
+      if (productsMapping && Array.isArray(productsMapping)) {
+        productsMapping.forEach((mapping) => {
+          productProperties[getPropertyName(mapping.to, index + 1)] = item[mapping.from];
```

Comment:
Yes they are getting recalculated but they are not overiding the same value.
Reason for doing this is we added a piece of code rather than refactoring the whole function